pkg/kubelet/cm/dra/plugin/registration.go

```diff
@@ -53,6 +54,18 @@ type RegistrationHandler struct {
 	backgroundCtx context.Context
 	kubeClient    kubernetes.Interface
 	getNode       func() (*v1.Node, error)
+	wipingDelay   time.Duration
+
+	mutex sync.Mutex
+
+	// pendingWiping maps a plugin name to a cancel function for
+	// wiping of that plugin's ResourceSlices. Entries get added
+	// in DeRegisterPlugin and check in RegisterPlugin. If
+	// wiping is pending during RegisterPlugin, it gets canceled.
+	//
+	// Must use pointers to functions because the entries have to
+	// be comparable.
+	pendingWiping map[string]*context.CancelCauseFunc
```

Comment:
nit: wdyt of naming this `pendingWipes`?

Reply:
Ack, changed.

Reply:
@mrunalp: [pushed](https://github.com/kubernetes/kubernetes/compare/3b0a27af309f465fcd5a3b92439fe58e57c14a17..582b421393d0fad2ad4a83feba88977ac4434662), please take another look.

Reply:
/lgtm